code/game/objects/structures/morgue.dm

```diff
@@ -141,7 +141,7 @@ GLOBAL_LIST_EMPTY(bodycontainers) //Let them act as spawnpoints for revenants an
  */
 /obj/structure/bodycontainer/morgue
 	name = "morgue"
-	desc = "Used to keep bodies in until someone fetches them."
+	desc = "Used to keep bodies in until someone fetches them. Now includes a high-tech alert system."

```

Comment:
just let the examine explain it.

Reply:
Its alright with the new examine text. The tray description points out the "high tech" alert system (which is ironically just a beeping noise), and the examine text tells you how to toggle it.